docker/test/integration/features/steps/steps.py

```diff
@@ -428,6 +428,7 @@ def step_impl(context):
 @given("an MQTT broker is set up in correspondence with the PublishMQTT and ConsumeMQTT")
 def step_impl(context):
     context.test.acquire_container(context=context, name="mqtt-broker", engine="mqtt-broker")
+    context.test.start('mqtt-broker')
```

Comment:
We already have a "the MQTT broker is started" step for this, which is used in the `Retained message` scenario outline.

It would be better to do this the same way in all scenarios: either add "When/And the MQTT broker is started" to all scenarios and remove the `start` from here, or remove the "the MQTT broker is started" step completely.

Reply:
I've merged the steps you mentioned in https://github.com/apache/nifi-minifi-cpp/pull/1652/commits/21d6cf4284885ad0855d4f7cb89b76362a19769b